# HG changeset patch
# User Rik <rik@octave.org>
# Date 1377062623 25200
# Node ID 68bcac3c043a282da6261e53b9f0cbb40914dde8
# Parent  d858e02f2a620dd6344ca53449db2742e6767430
Correct inversion accidentally introduced in cset 87ba70043bfc.

* scripts/plot/area.m, scripts/plot/axis.m, scripts/plot/caxis.m,
scripts/plot/comet.m, scripts/plot/comet3.m, scripts/plot/compass.m,
scripts/plot/contour.m, scripts/plot/contour3.m, scripts/plot/contourf.m,
scripts/plot/cylinder.m, scripts/plot/ellipsoid.m, scripts/plot/errorbar.m,
scripts/plot/feather.m, scripts/plot/fill.m, scripts/plot/loglog.m,
scripts/plot/loglogerr.m, scripts/plot/mesh.m, scripts/plot/meshc.m,
scripts/plot/meshz.m, scripts/plot/pcolor.m, scripts/plot/pie.m,
scripts/plot/pie3.m, scripts/plot/plot.m, scripts/plot/plot3.m,
scripts/plot/polar.m, scripts/plot/private/__bar__.m,
scripts/plot/private/__ezplot__.m, scripts/plot/private/__stem__.m,
scripts/plot/quiver.m, scripts/plot/quiver3.m, scripts/plot/rectangle.m,
scripts/plot/ribbon.m, scripts/plot/rose.m, scripts/plot/scatter.m,
scripts/plot/scatter3.m, scripts/plot/semilogx.m, scripts/plot/semilogxerr.m,
scripts/plot/semilogy.m, scripts/plot/semilogyerr.m, scripts/plot/slice.m,
scripts/plot/sphere.m, scripts/plot/stairs.m, scripts/plot/surf.m,
scripts/plot/surfc.m, scripts/plot/surfl.m, scripts/plot/surfnorm.m,
scripts/time/datetick.m: check for (! isempty (hax)) before saving
current figure.

diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/area.m
--- a/scripts/plot/area.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/area.m	Tue Aug 20 22:23:43 2013 -0700
@@ -104,7 +104,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/axis.m
--- a/scripts/plot/axis.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/axis.m	Tue Aug 20 22:23:43 2013 -0700
@@ -136,7 +136,7 @@
   [hax, varargin, nargin] = __plt_get_axis_arg__ ("axis", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/caxis.m
--- a/scripts/plot/caxis.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/caxis.m	Tue Aug 20 22:23:43 2013 -0700
@@ -45,7 +45,7 @@
   [hax, varargin, nargin] = __plt_get_axis_arg__ ("caxis", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/comet.m
--- a/scripts/plot/comet.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/comet.m	Tue Aug 20 22:23:43 2013 -0700
@@ -58,7 +58,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/comet3.m
--- a/scripts/plot/comet3.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/comet3.m	Tue Aug 20 22:23:43 2013 -0700
@@ -60,7 +60,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/compass.m
--- a/scripts/plot/compass.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/compass.m	Tue Aug 20 22:23:43 2013 -0700
@@ -100,7 +100,7 @@
   [r, p] = cart2pol (x, y);
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/contour.m
--- a/scripts/plot/contour.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/contour.m	Tue Aug 20 22:23:43 2013 -0700
@@ -64,7 +64,7 @@
   [hax, varargin] = __plt_get_axis_arg__ ("contour", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/contour3.m
--- a/scripts/plot/contour3.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/contour3.m	Tue Aug 20 22:23:43 2013 -0700
@@ -66,7 +66,7 @@
   [hax, varargin, nargin] = __plt_get_axis_arg__ ("contour3", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/contourf.m
--- a/scripts/plot/contourf.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/contourf.m	Tue Aug 20 22:23:43 2013 -0700
@@ -65,7 +65,7 @@
   [hax, varargin] = __plt_get_axis_arg__ ("contour", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/cylinder.m
--- a/scripts/plot/cylinder.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/cylinder.m	Tue Aug 20 22:23:43 2013 -0700
@@ -84,7 +84,7 @@
     zz = z;
   else
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/ellipsoid.m
--- a/scripts/plot/ellipsoid.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/ellipsoid.m	Tue Aug 20 22:23:43 2013 -0700
@@ -75,7 +75,7 @@
     zz = z;
   else
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/errorbar.m
--- a/scripts/plot/errorbar.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/errorbar.m	Tue Aug 20 22:23:43 2013 -0700
@@ -131,7 +131,7 @@
   [hax, varargin] = __plt_get_axis_arg__ ("errorbar", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/feather.m
--- a/scripts/plot/feather.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/feather.m	Tue Aug 20 22:23:43 2013 -0700
@@ -99,7 +99,7 @@
        ytmp - u * arrowsize / 3];
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/fill.m
--- a/scripts/plot/fill.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/fill.m	Tue Aug 20 22:23:43 2013 -0700
@@ -73,7 +73,7 @@
   iargs = __find_patches__ (varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/loglog.m
--- a/scripts/plot/loglog.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/loglog.m	Tue Aug 20 22:23:43 2013 -0700
@@ -46,7 +46,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/loglogerr.m
--- a/scripts/plot/loglogerr.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/loglogerr.m	Tue Aug 20 22:23:43 2013 -0700
@@ -49,7 +49,7 @@
   [hax, varargin] = __plt_get_axis_arg__ ("loglogerr", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/mesh.m
--- a/scripts/plot/mesh.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/mesh.m	Tue Aug 20 22:23:43 2013 -0700
@@ -64,7 +64,7 @@
   [hax, varargin, nargin] = __plt_get_axis_arg__ ("mesh", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/meshc.m
--- a/scripts/plot/meshc.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/meshc.m	Tue Aug 20 22:23:43 2013 -0700
@@ -62,7 +62,7 @@
   [hax, varargin, nargin] = __plt_get_axis_arg__ ("meshc", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/meshz.m
--- a/scripts/plot/meshz.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/meshz.m	Tue Aug 20 22:23:43 2013 -0700
@@ -112,7 +112,7 @@
   endif
     
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/pcolor.m
--- a/scripts/plot/pcolor.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/pcolor.m	Tue Aug 20 22:23:43 2013 -0700
@@ -75,7 +75,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/pie.m
--- a/scripts/plot/pie.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/pie.m	Tue Aug 20 22:23:43 2013 -0700
@@ -61,7 +61,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/pie3.m
--- a/scripts/plot/pie3.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/pie3.m	Tue Aug 20 22:23:43 2013 -0700
@@ -62,7 +62,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/plot.m
--- a/scripts/plot/plot.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/plot.m	Tue Aug 20 22:23:43 2013 -0700
@@ -208,7 +208,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/plot3.m
--- a/scripts/plot/plot3.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/plot3.m	Tue Aug 20 22:23:43 2013 -0700
@@ -110,7 +110,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/polar.m
--- a/scripts/plot/polar.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/polar.m	Tue Aug 20 22:23:43 2013 -0700
@@ -50,7 +50,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/private/__bar__.m
--- a/scripts/plot/private/__bar__.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/private/__bar__.m	Tue Aug 20 22:23:43 2013 -0700
@@ -150,7 +150,7 @@
 
   if (nargout < 2)
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/private/__ezplot__.m
--- a/scripts/plot/private/__ezplot__.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/private/__ezplot__.m	Tue Aug 20 22:23:43 2013 -0700
@@ -428,7 +428,7 @@
 
   ## Now, actually call the correct plot function with valid data and domain.
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/private/__stem__.m
--- a/scripts/plot/private/__stem__.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/private/__stem__.m	Tue Aug 20 22:23:43 2013 -0700
@@ -38,7 +38,7 @@
       check_stem_arg (have_z, varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/quiver.m
--- a/scripts/plot/quiver.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/quiver.m	Tue Aug 20 22:23:43 2013 -0700
@@ -69,7 +69,7 @@
     print_usage ();
   else
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/quiver3.m
--- a/scripts/plot/quiver3.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/quiver3.m	Tue Aug 20 22:23:43 2013 -0700
@@ -72,7 +72,7 @@
     print_usage ();
   else
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/rectangle.m
--- a/scripts/plot/rectangle.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/rectangle.m	Tue Aug 20 22:23:43 2013 -0700
@@ -61,7 +61,7 @@
   [hax, varargin] = __plt_get_axis_arg__ ("rectangle", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/ribbon.m
--- a/scripts/plot/ribbon.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/ribbon.m	Tue Aug 20 22:23:43 2013 -0700
@@ -77,7 +77,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/rose.m
--- a/scripts/plot/rose.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/rose.m	Tue Aug 20 22:23:43 2013 -0700
@@ -91,7 +91,7 @@
 
   if (nargout < 2)
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/scatter.m
--- a/scripts/plot/scatter.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/scatter.m	Tue Aug 20 22:23:43 2013 -0700
@@ -76,7 +76,7 @@
     print_usage ();
   else
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/scatter3.m
--- a/scripts/plot/scatter3.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/scatter3.m	Tue Aug 20 22:23:43 2013 -0700
@@ -73,7 +73,7 @@
     print_usage ();
   else
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/semilogx.m
--- a/scripts/plot/semilogx.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/semilogx.m	Tue Aug 20 22:23:43 2013 -0700
@@ -46,7 +46,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/semilogxerr.m
--- a/scripts/plot/semilogxerr.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/semilogxerr.m	Tue Aug 20 22:23:43 2013 -0700
@@ -51,7 +51,7 @@
   [hax, varargin] = __plt_get_axis_arg__ ("semilogxerr", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/semilogy.m
--- a/scripts/plot/semilogy.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/semilogy.m	Tue Aug 20 22:23:43 2013 -0700
@@ -46,7 +46,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/semilogyerr.m
--- a/scripts/plot/semilogyerr.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/semilogyerr.m	Tue Aug 20 22:23:43 2013 -0700
@@ -51,7 +51,7 @@
   [hax, varargin] = __plt_get_axis_arg__ ("semilogyerr", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/slice.m
--- a/scripts/plot/slice.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/slice.m	Tue Aug 20 22:23:43 2013 -0700
@@ -134,7 +134,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/sphere.m
--- a/scripts/plot/sphere.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/sphere.m	Tue Aug 20 22:23:43 2013 -0700
@@ -72,7 +72,7 @@
     zz = z;
   else
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
     unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/stairs.m
--- a/scripts/plot/stairs.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/stairs.m	Tue Aug 20 22:23:43 2013 -0700
@@ -75,7 +75,7 @@
       [h, xs, ys] = __stairs__ (false, varargin{:});
     else
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
       unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/surf.m
--- a/scripts/plot/surf.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/surf.m	Tue Aug 20 22:23:43 2013 -0700
@@ -63,7 +63,7 @@
   [hax, varargin] = __plt_get_axis_arg__ ("surf", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/surfc.m
--- a/scripts/plot/surfc.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/surfc.m	Tue Aug 20 22:23:43 2013 -0700
@@ -65,7 +65,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/surfl.m
--- a/scripts/plot/surfl.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/surfl.m	Tue Aug 20 22:23:43 2013 -0700
@@ -137,7 +137,7 @@
   endif
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/plot/surfnorm.m
--- a/scripts/plot/surfnorm.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/plot/surfnorm.m	Tue Aug 20 22:23:43 2013 -0700
@@ -115,7 +115,7 @@
 
   if (nargout == 0)
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
    unwind_protect
diff -r d858e02f2a62 -r 68bcac3c043a scripts/time/datetick.m
--- a/scripts/time/datetick.m	Tue Aug 20 20:55:05 2013 -0700
+++ b/scripts/time/datetick.m	Tue Aug 20 22:23:43 2013 -0700
@@ -36,7 +36,7 @@
   [hax, varargin, nargin] = __plt_get_axis_arg__ ("datetick", varargin{:});
 
   oldfig = [];
-  if (isempty (hax))
+  if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   if (isempty (hax))
